kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/subscribable/SubscribableKafkaMessageSource.java

```diff
@@ -224,7 +258,7 @@ public void close() {
      */
```

Comment:
The `Builder`'s class-level JavaDoc should be adjusted to no longer fixate on a "list of topics".

Reply:
I believe, I found the occurrences of list of topics in the latest push.

Reply:
You have, and it looks good to me. Thanks! I'll resolve this comment.